src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -141,31 +148,47 @@ private async Task ApplyContext(HostAssignmentContext assignmentContext)
                 await DownloadAsync(zipUri, filePath);
 
                 _logger.LogInformation($"Extracting files to '{options.ScriptPath}'");
+
+                var stopwatch = Stopwatch.StartNew();
```

Comment:
@fabiocav I recall seeing an IDisposable `Measure` object for this pattern? Maybe it wasn't in this project, but just wanna make sure since There are few places in here we would like to log the time they take and having a using pattern for that would make make the code nicer.

Reply:
Yes we have a metrics logger that we use for logging latency events, e.g. [here](https://github.com/Azure/azure-functions-host/blob/e83723aa6a581e3559f54b8d7b4d4f796420004b/src/WebJobs.Script/Host/ScriptHost.cs#L269)

Reply:
That this guy ^^ said.

Reply:
Done.